docs/reference/modules/cross-cluster-search.asciidoc

```diff
@@ -372,3 +372,16 @@ image:images/ccs/ccs-dont-min-roundtrip-shard-results.svg[]
 final results in the {ccs} response.
 +
 image:images/ccs/ccs-min-roundtrip-client-response.svg[]
+
+[float]
+[[ccs-supported-configurations]]
+==== Supported configurations
+
+<<gateway-nodes-selection, Generally>> a gateway node can search remote
+clusters that are one major version ahead or behind the gateway node's version.
+Gateways nodes can also search remote clusters that are being
+<<rolling-upgrades, upgraded>> so long as both the "upgrade from" and
+"upgrade to" version are compatible with the gateway node.
+
```

Comment:
```suggestion
For example, a gateway node running {es} 6.0 can search a remote cluster running {es} 7.0, but cannot search a remote cluster running {es} 7.1.

```

Reply:
I think that an explicit example is helpful to clarify the previous statement. Something along these lines ☝️